docs/cloud/elementary-managed.mdx

```diff
@@ -2,11 +2,72 @@
 title: "Elementary managed"
```

Comment:
Maybe upper-case the m? _Elementary Managed_.
Unlike our other pages, this is a name of a product so should be capitalized in my opinion.

Reply:
Yes